### PR TITLE
Script improvements

### PR DESCRIPTION
**Pull request by skycommand**
- ~~**Script:** Marked each snippet as "PowerShell" instead of plain text. Both GitHub and Microsoft Docs can highlight PowerShell syntax.~~ (✔ Someone else applied this item to the public branch.)
- **Script:** Added `#Requires -RunAsAdministrator` because the `Mount-WindowsImage` cmdlet needs it.
- **Script:** Eliminated the path-concatenating code-spaghetti. A mixture of different string-concatenating features was used alongside `Join-Path`! As a result, reading, interpreting, and adopting the path section of the script was a total nightmare.
- **Script:** Replaced all instances of "`Write-Host`" with "`Write-Output`". This is a serious change, as PowerShell scripts do not always run attended. Per PowerShell guidelines, `Write-Host` is a last-resort cmdlet. Where possible, `Write-Output` should be used instead.
- **Script:** Changed `return "{0:HH:mm:ss}" -f (Get-Date)` into `return "{0:HH:mm:ss}" -f [DateTime]::Now`. It seems not everyone knows that Get-Date's output can be customized, and not always via its parameters. Hence, the former could have unintended consequences. It is also possible to write `Get-Date -Format "HH:mm:ss"` which is neither superior nor inferior. (Well, maybe it costs a few more CPU ticks...)
- **Markdown Linter:** Replaced inline HTML with Markdown
- **Markdown Linter:** Inserted the missing line breaks before each heading

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@skycommand Can you resolve the merge conflicts in the meantime when the PR is under review? Thank you.

**Comment by skycommand** [comment]
Done.

File: windows/deployment/update/media-dynamic-update.md [hunk 1]
```diff
@@ -96,7 +95,6 @@ Optional Components, along with the .NET feature, can be installed offline, howe
 
 These examples are for illustration only, and therefore lack error handling. The script assumes that the following packages is stored locally in this folder structure:
```
Comment (JohanFreelancer9) [inline windows/deployment/update/media-dynamic-update.md:96]:
```suggestion
These examples are for illustration only, and therefore lack error handling. The script assumes that the following packages are stored locally in this folder structure:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - not related to this PR, but one additional correction, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by skycommand** [comment]
Hi, @SteveDiAcetis. I hate to bother you but I think it's about time I did. Are you too busy or have you forgotten about this one? This PR has passed the review.

**Review by illfated — APPROVED** [review]
- [X] **Regular source file view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8342/files
- [X] **Hide whitespace changes:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8342/files?diff=unified&w=1 (shows effective changes)
- [X] **Rich diff view:** https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8342/files?short_path=76a5394#diff-76a5394da68b5ac8f13f5bbea3b030741192a71bd15c2b86ed189edb6d558acb (publish preview)

**Comment by illfated** [comment]
#sign-off

**Comment by jaimeo** [comment]
@SteveDiAcetis can you drop a comment here if you approve of the edits?

**Comment by SteveDiAcetis** [comment]
@skycommand, thanks for these changes. I do not have permission to approve, but do accept these changes. @jaimeo, please merge - thanks in advance.

**Pull request merged**